dandi/metadata.py

```diff
@@ -455,11 +455,22 @@ def extract_sex(metadata: dict) -> Optional[models.SexType]:
 def extract_strain(metadata: dict) -> Optional[models.StrainType]:
     value = metadata.get("strain", None)
     if value is not None and value != "":
+        # Don't assign cell lines to strain
+        if value.lower().startswith("cellline:"):
+            return None
         return models.StrainType(name=value)
     else:
         return None
 
 
+def extract_cellLine(metadata: dict) -> Optional[str]:
+    value = metadata.get("strain", None)
+    if value is not None and value != "" and value.lower().startswith("cellline:"):
```

Comment:
simplify and robustify?

```suggestion
    if isinstance(value, str) and value.lower().startswith("cellline:"):
```

since `.lower()` is safe only if it is a string... if just to simplify, could be:

```suggestion
    if value and value.lower().startswith("cellline:"):
```